data-pipeline-flink/user-cache-updater-2.0/src/main/scala/org/sunbird/dp/usercache/util/UserMetadataUpdater.scala

```diff
@@ -0,0 +1,254 @@
+      val reportInfoMap: mutable.Map[String, AnyRef] = if (updatedMap.get("iscustodianuser").get == true) getCustodianUserInfo(updatedMap, metrics, userId) else getStateUserInfo(updatedMap, metrics, userId)
+
+      logger.info(s"User details ( $userId ) are fetched from the db's and updating the redis now.")
+      reportInfoMap
+    } else {
+      logger.info(s"Skipping the event update from databases since event Does not have user properties or user sigin in type is Anonymous ")
+      userCacheData.asInstanceOf[mutable.Map[String,AnyRef]]
+    }
+  }
+
+  def getCustodianRootOrgId(): String = {
+    val custRootOrgIdQuery = QueryBuilder.select("value").from(config.keySpace, config.systemSettingsTable)
+      .where(QueryBuilder.eq("id", "custodianRootOrgId")).and(QueryBuilder.eq("field", "custodianRootOrgId")).toString
+    val custRootOrgId = cassandraConnect.findOne(custRootOrgIdQuery)
+    custRootOrgId.getString("value")
+  }
+
+  def readFromCassandra(keyspace: String, table: String, clause: Clause, metrics: Metrics): util.List[Row] = {
+    var rowSet: util.List[Row] = null
+    val query = QueryBuilder.select.all.from(keyspace, table).where(clause).toString
+    rowSet = cassandraConnect.find(query)
+    if (null != rowSet && !rowSet.isEmpty) {
+      metrics.incCounter(config.dbReadSuccessCount)
+      rowSet
+    } else {
+      metrics.incCounter(config.dbReadMissCount)
+      new util.ArrayList[Row]()
+    }
+
+  }
+
+  def extractUserMetaData(userDetails: util.List[Row]): mutable.Map[String, AnyRef] = {
+    val result: mutable.Map[String, AnyRef] = mutable.Map[String, AnyRef]()
+    if (null != userDetails && !userDetails.isEmpty) {
+      val row: Row = userDetails.get(0)
+      val columnDefinitions = row.getColumnDefinitions()
+      val columnCount = columnDefinitions.size
+      for (i <- 0 until columnCount) {
+        result.put(columnDefinitions.getName(i), row.getObject(i))
+      }
+    }
+    val framework = result.get("framework").getOrElse(new util.LinkedHashMap()).asInstanceOf[util.LinkedHashMap[String, List[String]]]
+    if(!framework.isEmpty)
+     {
+       val board = framework.getOrDefault("board", List()).asInstanceOf[util.ArrayList[String]]
+       val medium = framework.getOrDefault("medium", List()).asInstanceOf[util.ArrayList[String]]
+       val grade = framework.getOrDefault("gradeLevel", List()).asInstanceOf[util.ArrayList[String]]
+       val subject = framework.getOrDefault("subject", List()).asInstanceOf[util.ArrayList[String]]
+       val id = framework.getOrDefault("id", List()).asInstanceOf[util.ArrayList[String]]
+       result.+=("board" -> board, "medium" -> medium, "grade" -> grade, "subject" -> subject, "framework" -> id)
+     }
+    result
+  }
+  def getFlattenFramework(framework: mutable.Map[String, List[String]]): Unit = {
+    println("framework " + framework)
+  }
```

Comment:
What is this method for?

Reply:
Removed.. Mistakenly committed the code.. Sorry